src/app/components/elements/OrderHistory.jsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import HistoryRow from "./OrderhistoryRow";
```

Comment:
this is being used https://github.com/steemit/condenser/pull/2060/files#diff-153a69ec7ac931570754a21a62d2b9bdL35